_posts/guides/2019-08-27-guide_the-pbr-shader.md

```diff
@@ -59,22 +63,72 @@ PBR
+```
+PBR
+{
+	$basetexture "example\pbr_asset_d"
+	$mraotexture "example\pbr_asset_mrao"
+	$bumpmap "example\pbr_asset_n"
+	
+	$envmap "env_cubemap"
+	$parallax "1"
+	$parallaxdepth "0.03"
+	$parallaxcenter "0.0"
+}
+```
+
+<div class="note info">
+    <p>
+        The Parallax option requires that a Heightmap is embedded into the Alpha Channel of the Normal Map.  
+        You can check out one way of doing so <a href="#texture-creation---designer">below</a>.
+    </p>
+</div>
+
+`$parallax` simply enables the Parallax effect.
+`$parallaxdepth` specifies how “deep” the parallax effect will go.
+`$parallaxcenter` adjusts the centre-point of the parallax effect, nudging it closer or further away from the surface.
+
+<div class="note warning">
+    <p>
+        Parallax Occlusion has its limitations and things to be wary of.<br><br>
+        
+        The Parallax Shader has a limited number of layers to create the depth effect.<br>
+        If you make the height value too intense, these layers become very easy to notice and ruin the effect, as shown below.<br><br>
+
+        <img src="/assets/images/pbr_guide/pbr_layer_example.jpg" alt="Visible layers of Parallax" style="display: block; margin: auto;"><br>
+        
+        Another thing you might notice is that the depth effect continues past the edge of the brush, which can look very strange as well, and is more noticeable on higher depth settings.
+    </p>
+</div>
+
 ## Texture Creation - Painter
 When starting a new Project in Painter, you should be fine using the standard ***PBR - Metal Roughness*** template.  
 
 <img src="\assets\images\pbr_guide\pbr_painter_template.jpg" alt="Painter Template" style="display: block; margin: auto;">
 
-However, the PBR Shader uses MRAO textures, so a custom export template is needed or the channels must be composited in some other program.
+However, the PBR Shader uses MRAO textures, so a custom export template is needed otherwise the channels must be packed together into a single texture elsewhere.
 
 You can try this export template, which creates the MRAO texture as well as an Emissive texture if one exists:  
 > [PBR Shader Template](/assets/PBR MRAO Emissive.spexp)
 
+<div class="note warning">
+    <p>
+        This template does not include the Heightmap in the Normal Map Alpha for Parallax.
+    </p>
+</div>
+
 It can go in the `Allegorithmic\Substance Painter\shelf\export-presets` folder.
 
 ## Texture Creation - Designer
-Creating the Base Colour and Normal aren't covered in the Guide, this section is just to show how to composite an MRAO texture.
+Creating the Input textures isn’t covered in the Guide - This section simply shows how to composite both the **MRAO** and **Normal** map with the **Height** textures correctly.
+
+You can create a **MRAO** texture by putting the **Metalness**, **Roughness**, **Ambient Occlusion**, and a White *Uniform Color* Node into an *RGBA Merge* Node.
```

Comment:
British/Canadian "colour" is used elsewhere in the guide. Can do this or convert everything to "color".
Also there is a lot of emphasis on the keywords here that aren't emphasized earlier in the guide. Maybe just emphasize the first mention?
```suggestion
You can create a **MRAO** texture by putting the **Metalness**, **Roughness**, **Ambient Occlusion**, and a White *Uniform Colour* Node into an *RGBA Merge* Node.
```